### DISCUSSION TITLE
Noob advice

### DISCUSSION THREAD

**Discussion by DavidBC71** [chadgeary/cloudblock#34] [Q&A] [answered]
Thanks for posting this project. Very interesting.

For various reasons I'm interesting in deploying this but I'm a little intimidated. I'm a computer user rather than a developer or coder and I worry that I may get in over my head and break something permanently. If you have the time I'd be grateful if you could answer a couple of simple (simplistic?) questions. If this isn't an appropriate forum for these types of questions I apologize. 

I have an OCI account.

1.   If I set this up does it mean that all my internet traffic from that point on goes through OCI?

2.    Is it like a commercial VPN service where I can switch it off?

3.    Will I be able to continue to use my commercial VPN (Private Internet Access)?

Thanks again for taking the time to share your project.

**Comment by chadgeary** [comment, marked as answer]
> If I set this up does it mean that all my internet traffic from that point on goes through OCI?

If you want it to. There’s a variable called vpn_traffic. By default it’s set to DNS traffic (website address lookups) only. It’s easy to change.

> Is it like a commercial VPN service where I can switch it off? 

Yep, the WireGuard VPN client is very easy to use and flip on or off.

> Will I be able to continue to use my commercial VPN (Private Internet Access)?

Yes, though simultaneously active VPNs can behave in unexpected ways.

Watch the deployment videos if you have not! Also it’s very unlikely you’ll break anything. Use discord chat if you run into trouble, I’ll be there.

**Reply by DavidBC71** [reply]
Thanks for prompt reply. I feel a bit more confident to try this now. Cheers. D
